stock_inventory_revaluation/views/stock_inventory_revaluation_view.xml

```diff
@@ -113,7 +113,7 @@
                     <field name="name"/>
                     <field name="revaluation_type"/>
                     <field name="document_date"/>
-                    <field name="user_id"/>
+                    <field name="create_uid"/>
```

Comment:
@JordiBForgeFlow Do you have an advice on this ? Why did you use a specific field for this ?

Reply:
@rousseldenis, @JordiBForgeFlow, Can we go ahead, and merge this PR?  This is the only unresolved comment.